docs/modules/configuring/con-configuring-kafkaservice-cipher.adoc

```diff
@@ -0,0 +1,31 @@
+[id='con-configuring-kafkaservice-cipher-{context}']
+= Controlling the TLS ciphers used
+
+Some cipher suites, which are enabled by default for interoperability purposes, are no longer consider secure.
+
+IMPORTANT: It is considered good practice to disable insecure cipher suites. You can only do this is the proxied Kafka cluster you want to connect to supports the cipher suites that you want to use.
```

Comment:
```suggestion
IMPORTANT: It is good practice to disable insecure cipher suites. Only do this if the proxied Kafka cluster supports the cipher suites you want to allow.
```

Reply:
Should this be a note or in the main body to introduce what we're showing in the config?